contributors/devel/kubemark-guide.md

```diff
@@ -202,9 +234,18 @@ Because there's no easy way of mocking other managers (e.g. VolumeManager), they
 are not supported in Kubemark (e.g. we can't schedule Pods with volumes in them
 yet).
 
-As the time passes more fakes will probably be plugged into HollowNodes, but
-it's crucial to make it as simple as possible to allow running a big number of
-Hollows on a single core.
+We currently plan to extend kubemark along the following directions:
```

Comment:
does kubemark script already support isolated deployment of master components? (put etcd, scheduler, api server onto different vms or different cgroups)